### DISCUSSION TITLE
Filter and text search - Enabling the feature

### DISCUSSION THREAD

**Discussion by JoseSebastianGarcia** [ElmahCore/ElmahCore#197] [Q&A]
Hello guys!

I saw in the readme that is possible to enable the filter & search in versions 2.2.0 or greater.
![image](https://github.com/user-attachments/assets/fd044e2e-2223-4df0-9873-d91c5be8d5e6)

How can I enable this feature?

Thank you,
Best regards,

**Comment by kenbailey** [comment]
@JoseSebastianGarcia, I was also wondering about this also, see discussion #187.

Any word @ElmahCore or @jafin regarding this project.

The https://github.com/jrsearles/Elmah.AspNetCore fork looks promising. I've been watching and it's been active, I haven't switched our code bases to it yet though.
